context.go

```diff
@@ -0,0 +1,10 @@
+package telebot
+
+const Default string = ""
```

Comment:
There we go, default handler is here, why would you need to introduce `Bot.DefaultHandler` then?